service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/application/HostedVespaApplication.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.service.monitor.internal.application;
+
+import com.yahoo.config.provision.ApplicationId;
+import com.yahoo.config.provision.Capacity;
+import com.yahoo.config.provision.ClusterSpec;
+import com.yahoo.config.provision.NodeType;
+import com.yahoo.config.provision.TenantName;
+import com.yahoo.component.Version;
+
+/**
+ * @author freva
+ */
+public abstract class HostedVespaApplication {
+
+    public static final TenantName TENANT_NAME = TenantName.from("hosted-vespa");
+
+    protected final ApplicationId applicationId;
```

Comment:
these can be private since they can be exposed via public accessor methods

Reply:
Fixed.